### PR TITLE
Pern on heroku

### PR DESCRIPTION
**Pull request by LeiaQuesada**
https://trello.com/c/EGdRvyEd/6-heroku-deployment

Hi, this is the initial deployment set-up for our PERN stack project, please let me know if it needs adjustment or isn't running for you 

here's the heroku URL: https://getmygarden.herokuapp.com/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,26 +78,29 @@ MyGarden is an application that allows anyone to easily plot an edible garden, b
...
-![plant recommendations based on user's zone](Wireframes/zone.png)
-![selected plant information](Wireframes/plantid.png)
-![add plant to user's garden](Wireframes/addplant.png)
-![user's garden summary of plant groupings and recommended articles](Wireframes/gardensummary.png)
-![recommendations for plant companions](Wireframes/compatiblitysuggestions.png)
-![container plant grouping screen](Wireframes/containercontents.png)
-![vegetable recommendations](Wireframes/vegetablerecommendations.png)
-![alternate screen for recommendations/search](Wireframes/alternaterecommendationscreen.png)
-![articles for general instructions](Wireframes/articles.png)
+## Wireframes:
```
Comment (Sophia-SR) [inline README.md:89]: This code looks wonderful !!!!

**Review by aedward8 — COMMENTED** [review]
These changes look amazing!!

**Comment by aedward8** [comment]
This is such a great code I  will have to approve the PR!

**Review by aedward8 — APPROVED** [review]
Amazing work!!

**Pull request merged**